consensus/bor/bor.go

```diff
@@ -1145,6 +1157,8 @@ func (c *Bor) fetchAndCommitSpan(
 
```

Comment:
That's a big piece of code to measure. Could you add intermediate points to measure?